docs/devguide/metricset-details.asciidoc

```diff
@@ -87,27 +87,21 @@ functionality of the metricset and `Fetch` method from the data mapping.
 [float]
 ==== fields.yml
 
-The `fields.yml` file is used for different purposes:
-
-* Creates the Elasticsearch template
-* Creates the Kibana index pattern configuration
-* Creates the Exported Fields documentation for the metricset
-
-To make sure the Elasticsearch template is correct, it's important to keep this file up-to-date
-with all the changes. There is a `fields.yml` file under `module/{module}/_meta/fields.yml` that contains
-the general top level structure for all metricsets. Normally you only need to modify the description in this file.
-
-Here an example for the `fields.yml` file from the MySQL module.
+You can find up to 3 different types of files named `fields.yml` in the beats repository for each metricbeat module:
 
+* `metricbeat/fields.yml`: Contains the definitions to create the Elasticsearch template, the Kibana index pattern configuration and the exported fields documentation for metricsets.To make sure the Elasticsearch template is correct, it's important to keep this file up-to-date with all the changes. Generally, you shouldn't touch this file manually because it's generated by some commands in the build environment.
```

Comment:
```suggestion
* `metricbeat/fields.yml`: Contains the definitions to create the Elasticsearch template, the Kibana index pattern configuration and the exported fields documentation for metricsets. To make sure the Elasticsearch template is correct, it's important to keep this file up-to-date with all the changes. Generally, you shouldn't touch this file manually because it's generated by some commands in the build environment.
```